src/main/resources/public/battleship.html

```diff
@@ -0,0 +1,368 @@
+<!doctype html>
+<html>
+
+	<head>
+		<meta charset="utf-8">
+		<title>Battleship 13</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+
```

Comment:
I wish I knew how to rename files in get, without these giant changes appearing...